app/(chat)/page.tsx

```
@@ -12,7 +13,7 @@ export default async function Page() {
 
   if (!modelIdFromCookie) {
     return (
-      <>
+      <DeployProvider accessToken={process.env.NEXT_PUBLIC_DEFAULT_ACCESS_TOKEN!}>
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)